docs/military/townsville.md

```diff
@@ -4,7 +4,82 @@
 
 --8<-- "includes/abbreviations.md"
 
-Reserved.
+## Positions
 
+| Name               | Callsign       | Frequency        | Login Identifier              |
+| ------------------ | -------------- | ---------------- | --------------------------------------|
+| Townsville Approach    | Townsville Approach   | 126.800         | TL_APP                                   |
+| Townsville ADC    | Townsville Tower  | 118.300         | TL_TWR        |
+| Townsville SMC    | Townsville Ground  | 121.800         | TL_GND        |
+| Townsville ACD    | Townsville Delivery  | 128.100         | TL_DEL       |
+| Townsville ATIS    |   | 133.500         | YBTL_ATIS       |
+
+## Airspace
+### TCU
+TL TCU owns the Class C and G airspace within 36 DME TL from `SFC` to `FL180`
+
+<figure markdown>
+![TL TCU/ADC Airspace Structure](img/tltcu.png){ width="2000" }
+  <figcaption>TL ADC/TCU Airspace Structure</figcaption>
+</figure>
+
+### ADC
+TL ADC owns the Class C arispace in the TL CTR (Extends to 7NM of the thresholds of 01 and 19.) from `SFC` to `A015`. This airsapce is designed to facilitate Helicopter Traffic and Ciruit Traffic.
+
+### Surveillance
```

Comment:
This section can be removed, only applies to Class D Towers (surveillance is assumed to be available to Class C towers already)